surfsense_backend/app/tasks/connectors_indexing_tasks.py

```
@@ -912,3 +915,192 @@ async def index_linear_issues(
+                    
+                    for message in messages:
+                        try:
+                            content = message.get("content", "")
+                            if not content:
+                                continue
+
+                            content_hash = generate_content_hash(content)
+                            existing_doc_by_hash_result = await session.execute(
+                                select(Document).where(Document.content_hash == content_hash)
+                            )
+                            existing_document_by_hash = existing_doc_by_hash_result.scalars().first()
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)